capstone/src/test/java/com/google/step/RestaurantTest.java

```diff
@@ -0,0 +1,88 @@
+// Copyright 2020 Google LLC
```

Comment:
I noticed that the servlets were moved into a servlet directory. Should the tests be moved too?

Reply:
I put the test in the capstone/src/test/... path instead of capstone/src/main/..., because if it's there maven will run the tests when you do "mvn test" and it will run the tests every time you build the project. It that's not something we want then I'll move it for sure but I thought it was kind of nice to have them run any time you build!

Reply:
Ah okay, I didn't realize that maven projects should have this structure. Sounds good, although should it be under a "data" directory to mirror the path of the file its testing?

Reply:
Yes, thank you, will add!